ramp-database/ramp_database/tools/submission.py

```diff
@@ -68,7 +69,7 @@ def add_submission(session, event_name, team_name, submission_name,
     submission = (session.query(Submission)
                          .filter(Submission.name == submission_name)
                          .filter(Submission.event_team == event_team)
-                         .one_or_none())
+                         .first())
```

Comment:
I would rather keep `one_or_none` then explicitly handle the None case. Otherwise .first() would return an empty list, if none match, right?

Actually, `None` is already handled below, why is this necessary then?